packages/parser/README.md

```diff
@@ -81,7 +81,7 @@ Enable parsing JSX when `true`. More details can be found [here](https://www.typ
 
 Default `undefined`.
 
-This option allows you to provide a path to your project's `tsconfig.json`. **This setting is required if you want to use rules which require type information**. You may want to use this setting in tandem with the `tsconfigRootDir` option below.
+This option allows you to provide a path to your project's `tsconfig.json`. **This setting is required if you want to use rules which require type information**. Relative paths are interpreted relative to the current working directory, so you may want to use this setting in tandem with the `tsconfigRootDir` option below.
```

Comment:
```
Relative paths are interpreted relative to the current working directory
```
that not true in all cases, 

Relative paths are relative to:
 - tsconfigRootDir if provided
 - or to cwd if tsconfigRootDir is not

Reply:
Slightly tweaked wording to better communicate this would be:

> Relative paths are interpreted relative to the current working directory if `tsconfigRootDir` is not set. If you intend on running ESLint from directories other than the project root, you should consider using `tsconfigRootDir`.

Reply:
Great wording, updated